ToolBox/AlgoSeekOptionsConverter/AlgoSeekOptionsConverter.cs

```diff
@@ -70,35 +72,43 @@ public AlgoSeekOptionsConverter(Resolution resolution, DateTime referenceDate, s
         public void Convert()
         {
             //Get the list of all the files, then for each file open a separate streamer.
-            var files = Directory.EnumerateFiles(_remote, _remoteMask);
-            Log.Trace("AlgoSeekOptionsConverter.Convert(): Loading {0} AlgoSeekOptionsReader for {1} ", files.Count(), _referenceDate);
+            var compressedRawDatafiles = Directory.EnumerateFiles(_remote, _remoteMask).Select(f => new FileInfo(f)).ToList();
+
+            Log.Trace("AlgoSeekOptionsConverter.Convert(): Loading {0} AlgoSeekOptionsReader for {1} ", compressedRawDatafiles.Count(), _referenceDate);
 
             //Initialize parameters
-            var totalLinesProcessed = 0L;
-            var totalFiles = files.Count();
+            var totalLinesProcessed = 0L;   
+            var totalFiles = compressedRawDatafiles.Count();
```

Comment:
should be able to use `Count` property instead of `Enumerable.Count` here since it's  list type.